ci-operator/step-registry/openshift/svt/regression/openshift-svt-regression-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+
+pushd /tmp
+
+git clone https://github.com/openshift/svt --depth=1
+pushd svt
+
+echo "========Start Regression test for $SCRIPT========"
+set +e
+
+# Do not change, for these custom tests we need to specify this specific version 
+export KUBE_BURNER_VERSION='1.7.6'
+
+# If SCRIPT is not specified, find the script by the TEST_CASE.
+if [[ $SCRIPT == "" ]]
+then
+    SCRIPT=$(find perfscale_regression_ci -name $TEST_CASE.sh)
```

Comment:
Add an env in ref.yaml for TEST_CASE

Reply:
I'm not clear on the TEST_CASE usage here, we'll have to specify a script. That was left over from the regression jenkins file.  I will remove